docs/reference/search/validate.asciidoc

```diff
@@ -14,13 +80,15 @@ PUT twitter/_bulk?refresh
 --------------------------------------------------
 // TESTSETUP
```

Comment:
```suggestion
```
This is causing the CI to fail. `// TESTSETUP` must be the first comment in a file.